src/OSPSuite.Core/Domain/ISimulation.cs

```diff
@@ -6,12 +6,5 @@ namespace OSPSuite.Core.Domain
    public interface ISimulation : ILazyLoadable, IAnalysable, IUsesObservedData, IModelCoreSimulation
    {
       IEnumerable<CurveChart> Charts { get; }
-
-      /// <summary>
-      ///    Name of all compounds used in the simulation
-      /// </summary>
-      IReadOnlyList<string> CompoundNames { get; }
```

Comment:
I am moving as much as possible to ModelCoreSimulation so that I don't have to repeat myself all the time.
ISimulation is now basicaly a IModelCoreSimulation with analysis and observed data. Nothing more